css/_uswds-theme-custom-styles.scss

```diff
@@ -1232,6 +1232,16 @@ code[class*="language-plaintext"] {
   }
 }
 
+.usa-prose .usa-button {
+  &#{$external-href} {
+    display: inline-block;
+
+    &::after {
+      display: initial;
+    }
+  }
+}
+
```

Comment:
After diving a bit deeper into this, I lean away from creating a new rule specific to `usa-prose` and instead lean towards narrowing the `display:none` style rule in [line 1218](https://github.com/uswds/uswds-site/blob/main/css/_uswds-theme-custom-styles.scss#L1218) to something like `nav .usa-button`. 

I think it is a cleaner solution for the code - that way we aren't turning it off everywhere then than turning it on again, and it _might_ cause fewer interactions in the future. Curious what you think!

Reply:
Resolved in https://github.com/uswds/uswds-site/pull/2318/commits/9aa629e47058dfcb48d1b91b6487c8fefc3ce9ca